.jenkins

```diff
@@ -32,7 +32,7 @@ pipeline {
                         dockerfile {
                             filename 'Dockerfile.nvhpc'
                             dir 'scripts/docker'
-                            label 'nvidia-docker && large_images'
+                            label 'nvidia-docker && volta'
```

Comment:
NOOOOO
The image is huge do not run that on other machine.  Just specify the correct architecture...